crates/hir-ty/src/display.rs

```diff
@@ -714,66 +758,130 @@ impl HirDisplay for Const {
     }
 }
 
+impl<'db> HirDisplay for crate::next_solver::Const<'db> {
```

Comment:
Here too switch the Chalk impl to use this under the hood.

Reply:
done.